tests/testthat/test-carehomes-support.R

```diff
@@ -464,20 +470,22 @@ test_that("Can compute initial conditions", {
                append(rep(0, 18), 10, after = 3))
   expect_equal(drop(initial_y$I_weighted),
                append(rep(0, 18), p$I_A_transmission * 10, after = 3))
-  expect_equal(initial_y$T_sero_pre[, 1, 1, ],
+  expect_equal(initial_y$T_sero_pre_1[, 1, 1, ],
+               append(rep(0, 18), 10, after = 3))
+  expect_equal(initial_y$T_sero_pre_2[, 1, 1, ],
                append(rep(0, 18), 10, after = 3))
   expect_equal(initial_y$T_PCR_pos[, 1, 1, ],
                append(rep(0, 18), 10, after = 3))
   expect_equal(initial_y$react_pos, 10)
 
-  ## 46 here, derived from;
+  ## 48 here, derived from;
   ## * 38 (S + N_tot)
   ## * 1 (prob_strain)
   ## * 1 (react_pos)
-  ## * 2 (N_tot2 + N_tot3)
+  ## * 3 (N_tot_sero_2 + N_tot_PCR + N_tot_sero_2)
```

Comment:
should this be (N_tot_sero_1 + N_tot_PCR + N_tot_sero_2) instead?

Reply:
Corrected, and also reordered for clarity